### PR TITLE
Fix bug:

### PR DESCRIPTION
**Pull request by anhmht**
#79 #80 #84 
- Using the value instead of enum data(SearchResult.jsx)
- Show a tag 'Chưa xác định' when tracking doesn't have user
- Using dropdown to implement 'Chọn khách hàng' button
screencapture-localhost-3000-search-2019-03-23-23_28_37

 - Hide 'Chọn khách hàng' drop down when tracking already have user
 - Remove method search from SearchForm.jsx, replace with calling publications in SearchResult.jsx to make real time when use 'chọn khách hàng' button
 - Change 'Thêm chuyến bay' button like customer list page, change 'Mã chuyến bay' to 'Mã'
screencapture-localhost-3000-flights-2019-03-23-23_29_39

### CODE REVIEW

**Comment by hoadinh2010** [comment]
@anhmht  
image

**Comment by hoadinh2010** [comment]
image

File: imports/ui/modules/layout/components/SearchPage/SearchResult.jsx [hunk 1]
```diff
@@ -139,4 +237,79 @@ const mapStateToProps = (state) => {
...
 };
 
-export default connect(mapStateToProps, null)(SearchResult);
+export default compose(
+  reduxForm({
+    form: REDUX_FORM_NAMES.SEARCH_FORM,
+  }),
+
+  connect(mapStateToProps, null),
+  withTracker((props) => {
```
Comment (hoadinh2010) [inline imports/ui/modules/layout/components/SearchPage/SearchResult.jsx]: I don't know what are you doing here? Need to explain

Comment (anhmht) [reply imports/ui/modules/layout/components/SearchPage/SearchResult.jsx]: I removed the method get Tracking, and using the publication to get tracking instead. My purpose is to make it real time when user update the customer (if the tracking in UNKNOWN condition of customter).

**Pull request merged**